src/clothing_mod.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+#ifndef CLOTHING_MOD_H
+#define CLOTHING_MOD_H
+
+#include <stddef.h>
+#include <string>
+#include <vector>
+
+#include "type_id.h"
+#include "string_id.h"
+#include "ui.h"
+#include "output.h"
+
+class JsonObject;
+class player;
+class item;
+
+struct mod_value {
+    std::string type;
```

Comment:
This should probably be an enumeration. That would make its usage much saver as it
- prevents spelling mistakes like `get_mod_val( "acit", it )` (note: your JSON documentation mentions that specific string as an allowed value, bu tin the code, you call the function with "acid").
- has better performance.

Reply:
changed to enumeration.